sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -75,7 +86,37 @@ export interface MicrosoftTeamsAppIdentifier {
   teamsAppId: string;
 
   /**
-   * The cloud that the Microsoft Temas App belongs to. If missing, the cloud is "public".
+   * The cloud that the Microsoft Teams App belongs to. If missing, the cloud is "public".
+   */
+  cloud?: "public" | "dod" | "gcch";
+}
+
+/**
+ * A Microsoft Teams Phone user who is using a Communication Services resource
```

Comment:
```suggestion
 * A Microsoft Teams Extension user who is using an Azure Communication Services resource
```

Reply:
Using same value as description from the new swagger:
"A Microsoft Teams Phone user who is using a Communication Services resource to extend their Teams Phone set up."